src/main/java/core/org/akaza/openclinica/dao/hibernate/EventCrfDao.java

```diff
@@ -16,9 +17,12 @@
 
     public List<EventCrf> findNonArchivedByStudyEventId(int study_event_id) {
         String query = "from " + getDomainClassName()
-                + " event_crf where event_crf.studyEvent.studyEventId = :studyeventid and event_crf.statusId not in (5,7)";
+                + " event_crf where event_crf.studyEvent.studyEventId = :studyeventid " +
+                "and (event_crf.removed = null or event_crf.removed !=:removed) and (event_crf.archived = null or event_crf.archived !=:archived) ";
```

Comment:
can this be simplified to "event_crf.removed != true) and (event_crf.archived != true)", since not true means either null or false?

Reply:
If I change the code to yours, getting the following error. Named parameter [archived] not set
	at org.hibernate.query.internal.QueryParameterBindingsImpl.verifyParametersBound(QueryParameterBindingsImpl.java:242)
	at org.hibernate.query.internal.AbstractProducedQuery.beforeQuery(AbstractProducedQuery.java:1245)
	at org.hibernate.query.internal.AbstractProducedQuery.list(AbstractProducedQuery.java:1336)
	at core.org.akaza.openclinica.dao.hibernate.EventCrfDao.findNonArchivedByStudyEventId(EventCrfDao.java:28)

Reply:
probably because you have the "hibernateQuery.setParameter("archived", true);" below

Reply:
I removed that line earlier , still got same error . 
Since we have removed and archived properties as object, and when the object is null , then event_crf.removed !=true  (null != true) somehow throwing,   Named parameter [removed] not set error

Reply:
Got it, let's keep this then.